generator.rb

```diff
@@ -0,0 +1,65 @@
+def is_vowel? letter
+  return ["a", "e", "i", "o", "u"].include?(letter)
+end
+
+def run_generator
+
+  puts "What is the first word to combine?"
+
+  a = gets.chomp
+
```

Comment:
Might make sense to separate getting user input and doing the work - this method ends up being pretty long. Maybe have them write a method that gets a word from the user?